Dockerfile

```diff
@@ -1,13 +1,10 @@
-FROM pypy:slim-bullseye
-
-RUN pip install -U poetry
-RUN poetry config virtualenvs.create false
+FROM python:3.12
 
 WORKDIR /loxsi
 
-COPY pyproject.toml poetry.lock ./
-RUN poetry install --only main
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
 
 COPY . .
 
-CMD [ "pypy3", "-m", "uvicorn", "--host", "0.0.0.0", "--port", "80", "main:app" ]
```

Comment:
- I still need to do a proper review of the code.

What is the reason to move away from pypy and poetry?
- pypy gives us some imploved performance
- poetry made it easy to manage the few dependencies we have and separate main from development dependencies

Reply:
- The performance gains from pypy is negligible compared to python 3.12.2
- Poetry got added in [b8e2990](https://github.com/12urenloop/Loxsi/pull/13/commits/b8e29905f67c65273fc1e02c3462deaede547fdb)